examples/led_deck.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+from crazyflow.control.control import Control
+from crazyflow.sim import Physics, Sim
+from crazyflow.sim.visualize import change_material
+
+
+def main():
+    """Spawn 25 drones in one world and activate led decks."""
+    n_worlds, n_drones = 1, 25
+    sim = Sim(
+        n_worlds=n_worlds,
+        n_drones=n_drones,
+        drone_model="cf21B_500",
+        control=Control.state,
+        physics=Physics.so_rpy,
+        device="cpu",
+    )
```

Comment:
The basic scripts made these kwargs explicit to show users the options we provide with the sim. In specialized examples such as this one, we can skip this. 

```suggestion
    sim = Sim(n_drones=25, drone_model="cf21B_500")
```

Reply:
Simplified.